File: translations/codenet/gpt_commented/C++2C/deepseek/Code/s984540390.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main() {
    long long ans = 0, mx = 0, sum = 0, mn = 100000000, cnt = 1;
    long long h;
    scanf("%lld", &h);

    while (h != 1) {
        if (h % 2 != 0) h--;
        h /= 2;
        cnt *= 2;
        ans += cnt;
    }

    printf("%lld\n", ans + 1);
}

File: translations/codenet/uncommented/C++2Go/deepseek/Code/s984540390.go
package main

import (
	"fmt"
)

func main() {
	var h int
	fmt.Scan(&h)
	ans, cnt := 0, 1
	for h != 1 {
		if h%2 != 0 {
			h--
		}
		h /= 2
		cnt *= 2
		ans += cnt
	}
	fmt.Println(ans + 1)
}

